FontAtlasProsessor.go

```diff
@@ -300,23 +308,36 @@ func (a *FontAtlas) rebuildFontAtlas() {
 	// Add extra fonts
 	for _, fontInfo := range a.extraFonts {
 		// Scale font size with DPI scale factor
-		if runtime.GOOS == windows {
-			fontInfo.size *= Context.GetPlatform().GetContentScale()
-		}
+		// TODO
+		//if runtime.GOOS == windows {
+		//	fontInfo.size *= Context.GetPlatform().GetContentScale()
+		//}
 
 		// Store imgui.Font for PushFont
 		var f imgui.Font
 		if len(fontInfo.fontByte) == 0 {
-			f = fonts.AddFontFromFileTTFV(fontInfo.fontPath, fontInfo.size, imgui.DefaultFontConfig, ranges.Data())
+			f = fonts.AddFontFromFileTTFV(
+				fontInfo.fontPath,
+				fontInfo.size,
+				0,
+				ranges.Data(),
+			)
 		} else {
-			f = fonts.AddFontFromMemoryTTFV(fontInfo.fontByte, fontInfo.size, imgui.DefaultFontConfig, ranges.Data())
+			f = fonts.AddFontFromMemoryTTFV(
+				unsafe.Pointer(&fontInfo.fontByte[0]),
+				int32(fontInfo.size),
+				0,
+				0, // TODO: in ref to imgui-go: `const DefaultFontConfig FontConfig = 0`
+				ranges.Data(),
+			)
 		}
 
 		a.extraFontMap[fontInfo.String()] = &f
 	}
 
-	fontTextureImg := fonts.TextureDataRGBA32()
-	Context.renderer.SetFontTexture(fontTextureImg)
+	// TODO
+	//fontTextureImg, _, _, _ := fonts.GetTextureDataAsRGBA32()
+	//Context.renderer.SetFontTexture(fontTextureImg)
```

Comment:
@AllenDang I have a problem with thisone - what is this SetFontTexture in cimgui? Should we export something from glfw?

Reply:
Yes, you should export something from gl, the renderer.

Reply:
frangly, it works without this...